tests/behat/features/templates/projects-engage-auth.feature

```diff
@@ -14,6 +14,12 @@ Feature: test projects/engagements page
     When I fill in "edit-search--2" with "test"
     And I wait 4 seconds
     Then I should see "Test"
+    Then I should see "test-project-title"
+    Then I should see "Project Institution"
```

Comment:
Looks like the title is test-create-project-title
![image](https://user-images.githubusercontent.com/539763/219458719-393e554d-f512-4a73-9925-fd0de76efab3.png)